### PR TITLE
Migrate XtextAntlrGeneratorFragment

### PR DESCRIPTION
**Pull request by oehme**

### CODE REVIEW

File: plugins/org.eclipse.xtext.xtext.generator/src/org/eclipse/xtext/xtext/generator/parser/antlr/XtextAntlrGeneratorFragment2.xtend [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+		val rtBindings = new GuiceModuleAccess.BindingFactory()
+			.addTypeToType(IParser.typeRef, grammar.parserClass)
+			.addTypeToType(ITokenToStringConverter.typeRef,AntlrTokenToStringConverter.typeRef)
+			.addTypeToType(IAntlrTokenFileProvider.typeRef, grammar.antlrTokenFileProviderClass)
+			.addTypeToType(Lexer.typeRef, grammar.lexerClass)
+			.addTypeToType(ITokenDefProvider.typeRef, AntlrTokenDefProvider.typeRef)
+			.addTypeToProviderInstance(grammar.lexerClass, '''�LexerProvider�.create(�grammar.lexerClass�.class)''')
+			.addConfiguredBinding("RuntimeLexer", '''
+					binder.bind(�Lexer�.class)
+					.annotatedWith(�Names�.named(�LexerBindings�.RUNTIME))
```
Comment (szarnekow) [inline plugins/org.eclipse.xtext.xtext.generator/src/org/eclipse/xtext/xtext/generator/parser/antlr/XtextAntlrGeneratorFragment2.xtend]: Cosmetics: please increase indentation in multiline expressions, e.g on line 2 and 3 for the `binder....class); `

File: web/org.eclipse.xtext.web.example.statemachine/src-gen/org/eclipse/xtext/web/example/statemachine/parser/antlr/internal/InternalStatemachine.g [hunk 2]
```diff
@@ -9,7 +9,6 @@ grammar InternalStatemachine;
```
Comment (szarnekow) [inline web/org.eclipse.xtext.web.example.statemachine/src-gen/org/eclipse/xtext/web/example/statemachine/parser/antlr/internal/InternalStatemachine.g]: Eventually proper indentation in the *.g files :)

**Comment by oehme** [comment]
Jenkins test

**Comment by oehme** [comment]
Jenkins test

**Pull request merged**